experimental/uefi/loader/src/crosvm.rs

```diff
@@ -75,4 +131,8 @@ impl Vmm for Crosvm {
         self.instance.start_kill()?;
         self.wait().await
     }
+
+    fn create_comms_channel(&self) -> Result<Box<dyn ReadWrite>> {
+        Ok(Box::new(VsockStreamChannel { stream: None }))
```

Comment:
Why is the stream `None`?

Reply:
Ah, I think it says in line 47, maybe it is worth to repeat/link here? Or maybe rename `create_comms_channel_without_stream`?

Reply:
Added a comment. The function is part of a trait, so the other implementation on QEMU would have to be renamed. On QEMU it is connected before booting the VM, so the name change would not make sense there.